### ISSUE TITLE
Food assistance should be asked for the entire household, not per member

### ISSUE THREAD

**Issue by vrajmohan** [18F/eligibility-rules-service#208]
We are changing the way we ask for food assistance. We will no longer ask about individuals, but ask about the household.

**AC:** 

- [ ] Change the input to yes/no radio buttons (Adam)
- [ ] Remove the question description (it is no longer needed) (Adam)
- [ ] Update JSON schema in registration service (Vraj)

---
*Original description* 

The way that we currently ask about food assistance is that we ask about each person... 

![image (3)](https://user-images.githubusercontent.com/2066204/55029579-a6616e00-4fc7-11e9-8ec6-514c80b1847f.png)

...rather than as a group, "Does person 1, Person 2, or Person 3 receive food assistance? Yes or No"

As we understand it, we only really need to know about the household as a group, because if anyone did receive food assistance the household would be ineligible.

But I also think, for example, if a child who live with a Mom and Stepdad gets food assistance through the biological father, then really, that child could be removed from the household and the two adults could become eligible. 

So my question is: should we make this a Yes or No question or should we keep it with individual checkboxes?

**Comment by edmullen** [comment]
The decision has been made to change this to a household-level yes or no question. So we should no longer ask about the individual household members as checkboxes, but use or standard yes/no radio buttons. 

The content document has been updated to reflect this.

**Comment by edmullen** [comment]
Please do not start this issue until one outstanding issue is resolved.
